src/server/pps/server/parallelism_test.go

```diff
@@ -20,30 +19,28 @@ func wrap(t testing.TB, ps *pps.ParallelismSpec) *pps.PipelineInfo {
 }
 
 func TestGetExpectedNumWorkers(t *testing.T) {
-	kubeClient := tu.GetKubeClient(t)
-
 	// An empty parallelism spec should default to 1 worker
-	workers, err := getExpectedNumWorkers(kubeClient, wrap(t,
+	workers, err := getExpectedNumWorkers(wrap(t,
```

Comment:
I can't find where this changed.  Why don't we need the kubeClient anymore?

Reply:
The function no longer uses it. I think we used to determine default parallelism by the number of nodes